wp-content/themes/core/assets/js/editor/block-animations/settings/easing.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @module easing
+ *
+ * @description pulls animation easing settings from theme.json or sets default settings
+ *
+ * theme.json settings:
+ *
+ * "animationEasing": [
+ * 		{ "label": "Ease In", "value": "ease-in" },
+ * 		{ "label": "Ease Out", "value": "ease-out" }
+ * ],
+ */
+
+import { __ } from '@wordpress/i18n';
+import themeJson from '../../../../../theme.json';
+
+const easing = themeJson?.settings?.animationEasing ?? [
+	{
+		label: __( 'Ease Out Sine', 'tribe' ),
+		value: 'cubic-bezier(0.390, 0.575, 0.565, 1.000)',
```

Comment:
Just a thought: I know we've got a PR to solve the use of this value in inline style blocks, but I wonder if it would be best to move these to global custom properties that could be used anywhere. I.e. - replace the values here with something like `--easing--ease-out-sine` and then populate the `cubic-bezier()` values in the global css var. Naming them as vars makes it easier to understand the values when they appear in code, too. ;)